src/linear_algebra/bilinear_form.lean

```diff
@@ -393,6 +418,32 @@ end
 
 section is_basis
 
+/-- A set of orthogonal vectors `v` with respect to some bilinear form `B` is linearly independent
+  if for all `i`, `B (v i) (v i) ≠ 0`. -/
+lemma linear_independent_of_is_Ortho
+  {n : Type w} {B : bilin_form K V} {v : n → V}
+  (hv₁ : B.is_Ortho v) (hv₂ : ∀ i, ¬ B.is_ortho (v i) (v i)) :
+  linear_independent K v :=
```

Comment:
I think this is probably accidental -- instead of putting `linear_independent_of_is_Ortho` in the same section as the other `is_ortho`/`is_Ortho` lemmas, you put it just after the start of the next section:
```lean
end

section is_basis
```
Can you move it up 3 lines?